UM-USElab/gradecraft-development
# 2746 Add email to send to instructors on course creation

### Status
**READY**

### Description
When a new course membership with the role of staff is created, we would like to send the user support materials. Update the course_membership create and update methods so that when a staff member is set, they receive an email. Talk to Evan regarding what the context of the should be.

### Related PRs
N/A


### Todos


### Deploy Notes
N/A

### Gem dependencies
N/A

### Migrations
YES

### Steps to Test or Reproduce
As an admin, create a new 'staff' course membership. Then check that new user's email and verify that the user received the new resources email.

### Impacted Areas in Application
List general components of the application that this PR will affect:

* User Create

======================
Closes #2746
